### DISCUSSION TITLE
Add Tri-state chips and busses

### DISCUSSION THREAD

**Discussion by megarcher** [DigitalLogicSimCommunity/Digital-Logic-Sim-CE#69] [Ideas]
Hi, 
It would be great if we could have tri state buffers and buses, like the new DLS from Sebastian has. This version has bundled cables, displays and key inputs, so I would rather use this version, but tri-state buffers lake from the game.
Could someone tell me how to add new built-in chips in unity too? I cannot find the script that loads the chips and can therefore not load my custom built-in chip. 
Thanks!

**Comment by jafr-odoo** [comment]
it would add so much quality of life, same for a 7-segment display or some simple way to interpret binary in decimal, hell a great idea would be a built-in chip that allows for writing a custom script.
